src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -82,84 +92,85 @@ void BgMizuMovebg_Init(Actor* thisx, PlayState* play) {
+    ((BgMizuMovebg*)thisx)->dlist = sDLists[MOVEBG_TYPE(thisx->params)];
     DynaPolyActor_Init(&((BgMizuMovebg*)thisx)->dyna, DYNA_TRANSFORM_POS);
-    CollisionHeader_GetVirtual(D_8089EB70[MOVEBG_TYPE(thisx->params)], &colHeader);
+    CollisionHeader_GetVirtual(sColHeaders[MOVEBG_TYPE(thisx->params)], &colHeader);
     ((BgMizuMovebg*)thisx)->dyna.bgId = DynaPoly_SetBgActor(play, &play->colCtx.dyna, thisx, colHeader);
 
     type = MOVEBG_TYPE(thisx->params);
     switch (type) {
-        case 0:
-            temp = waterBoxes[2].ySurface + 15.0f;
-            if (temp < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
+        case MIZUMOVEBG_TYPE_FLOAT_PLATFORM_WEST:
+            targetPosY = waterBoxes[2].ySurface + 15.0f;
+            if (targetPosY < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
                 thisx->world.pos.y = ((BgMizuMovebg*)thisx)->homeY - 700.0f;
             } else {
-                thisx->world.pos.y = temp;
+                thisx->world.pos.y = targetPosY;
             }
-            ((BgMizuMovebg*)thisx)->actionFunc = func_8089E318;
+            ((BgMizuMovebg*)thisx)->actionFunc = BgMizuMovebg_UpdateMain;
             break;
-        case 1:
-            temp = waterBoxes[2].ySurface + 15.0f;
-            if (temp < ((BgMizuMovebg*)thisx)->homeY - 710.0f) {
+        case MIZUMOVEBG_TYPE_FLOAT_PLATFORM_OUTSIDE_CENTER_PILLAR:
+            targetPosY = waterBoxes[2].ySurface + 15.0f;
+            if (targetPosY < ((BgMizuMovebg*)thisx)->homeY - 710.0f) {
                 thisx->world.pos.y = ((BgMizuMovebg*)thisx)->homeY - 710.0f;
             } else {
-                thisx->world.pos.y = temp;
+                thisx->world.pos.y = targetPosY;
             }
-            ((BgMizuMovebg*)thisx)->actionFunc = func_8089E318;
+            ((BgMizuMovebg*)thisx)->actionFunc = BgMizuMovebg_UpdateMain;
             break;
-        case 2:
-            temp = waterBoxes[2].ySurface + 15.0f;
-            if (temp < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
+        case MIZUMOVEBG_TYPE_FLOAT_PLATFORM_INSIDE_CENTER_PILLAR:
+            targetPosY = waterBoxes[2].ySurface + 15.0f;
+            if (targetPosY < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
                 thisx->world.pos.y = ((BgMizuMovebg*)thisx)->homeY - 700.0f;
             } else {
-                thisx->world.pos.y = temp;
+                thisx->world.pos.y = targetPosY;
             }
-            ((BgMizuMovebg*)thisx)->actionFunc = func_8089E318;
+            ((BgMizuMovebg*)thisx)->actionFunc = BgMizuMovebg_UpdateMain;
             break;
-        case 3:
-            thisx->world.pos.y = ((BgMizuMovebg*)thisx)->homeY + D_8089EB40[func_8089DC30(play)];
-            ((BgMizuMovebg*)thisx)->actionFunc = func_8089E318;
+        case MIZUMOVEBG_TYPE_DRAGON_STATUE_BOSS_ROOM:
+            thisx->world.pos.y =
+                ((BgMizuMovebg*)thisx)->homeY + sDragonStatue1OffsetPosY[BgMizuMovebg_GetDragonStatue1OffsetIndex(play)];
```

Comment:
run formatter

also it seems like "dragon statue 1" can be changed to "dragon statue boss room" in the data and function names?